.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description, Motivation and Context
+
+Please include a summary of the change or what problem does it solve. Please also include relevant motivation and context.
+
+<!--- If it relates to an open issue, please link to the issue here.
+e.g.
+[issue 414](https://github.com/replicatedhq/troubleshoot.sh/issues/414)
+-->
+
+## Checklist
+
+- [ ] The impact of the change checked.
```

Comment:
Can we re-word this, it's not particularly clear as to what it means.

Reply:
I rephrase it into `New and existing tests pass locally with the changes.` Maybe this is more clear.